lib/services/azuredocdb/index.js

```diff
@@ -30,8 +30,8 @@ Handlers.provision = function(log, params, next) {
   if (!cp.allValidatorsSucceed()) {
     var reply = Reply(HttpStatus.INTERNAL_SERVER_ERROR);
     reply.value.description = 'Input validators failed.';
-    var err = new Error('Validation failed.');
-    err.code = HttpStatus.INTERNAL_SERVER_ERROR;
+    var err = new Error('Parameter validation failed.  Did you supply a DocDb parameters file?');
```

Comment:
In my high school days when I learned to type, they said "one blank after commas, two after periods".  Are they teaching something different these days?